environment-dev.yaml

```diff
@@ -16,3 +16,7 @@ dependencies:
   - pytest
   - diagrams
   - jhub-client
+
+  # deployment/integration tests dependencies
+  - dask-gateway
+  - python-kubernetes
```

Comment:
I think this should be `kubernetes` not `python-kubernetes` (which is very old, and doesn't work with the tests).

But also, note that `pip install -e .[dev]` does not make use of this environment-dev.yaml file, so really "kubernetes" and "dask-gateway" need to be added to the dev list in setup.py

Reply:
The correct pip package is `kubernetes`, but the conda's `kubernetes` package didn't work for me yesterday and `python-kubernetes` seems to work, either ways I'll update the dev dependency.